static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -118,6 +118,8 @@ $ ls -i .dvc/cache/ec/
 $ du -sh .
  41M .
 ```
+Do note that `ls -i` prints prints the index number of each file and index 
+number for `data/Posts.xml.zip` and `.dvc/cache/ec` remains same always.
```

Comment:
`.dvc/cache/ec` is a directory, it contains a file `88519f8465218abb23ce0e0e8b1384` that has the same inode as `data/Posts.xml.zip`

Reply:
`remains same always` -> `remained same`